lib/templates/kubernetes/README.md

```diff
@@ -0,0 +1,24 @@
+#### Deploy wheel on Kubernetes
+
+Create postgres db and service.
+
+`kubectl create -f db-deployment.yml -f db-service.yml`
+
+Create configmap for database.yml.
+
+`kubectl create -f database-configmap.yml`
+
+Create web/puma deployment and service.
+
+`kubectl create -f web-deployment.yml -f web-service.yml`
+
+Create background/delayed_job deployment and service.
+
+`kubectl create -f background-deployment.yml -f background-service.yml`
+
+Get application endpoint.
```

Comment:
Need links to how to setup this AWS integration before we begin this whole block.